cwl/src/test/resources/three_step.cwl

```diff
@@ -1,47 +1,46 @@
 cwlVersion: v1.0
 class: Workflow
 inputs:
-- id: file:///Users/danb/wdl4s/r.cwl#pattern
+- id: pattern
```

Comment:
FMI, I'm assuming this only works because we re-SALAD this file before processing?

Reply:
yep